src/frontend/src/app/pagination/pagination.component.css

```diff
@@ -0,0 +1,45 @@
+.pagination {
```

Comment:
mobile responsive?

Reply:
Now there is no use of this css file. I will remove it.